tcipostman/__init__.py

```diff
@@ -0,0 +1,25 @@
+from flask_mail import Mail, Message
+from tcidatabase import models
+from tcidatabase import db
+
+
+def init_app(app):
+    Postman.app = app
+    Postman.mail = Mail(app)
+    db.signals.post_save.connect(Postman.send_client_token,
+                                 sender=models.SingleToken)
+
+
+class Postman:
+    app = None
+    mail = None
+
+    @classmethod
+    def send_client_token(klass, sender, document, created, **kwargs):
```

Comment:
I don't see when `sender` and I don't get the convention's names of the variables